projects/plugins/jetpack/extensions/shared/currencies.js

```diff
@@ -93,3 +98,40 @@ export function parseAmount( amount, currency ) {
 
 	return amount;
 }
+
+/**
+ * @param { string } postId - ID of the current post
+ * @param { string } connectURL - Stripe connect URL
+ * @returns { null | string } URL
+ */
+export function getConnectUrl( postId, connectURL ) {
```

Comment:
Tiny nit: the other functions in this file are to do with dealing with currencies. Would it be worth moving this function to its own `get-connect-url.js` file?